frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -4,12 +4,14 @@
 from msgpack import packb, unpackb
 
 from frontera.core.codec import BaseDecoder, BaseEncoder
+import six
+from six.moves import map
 
 
 def _prepare_request_message(request):
     def serialize(obj):
         """Recursively walk object's hierarchy."""
-        if isinstance(obj, (bool, int, long, float, basestring)):
+        if isinstance(obj, (bool, int, float, six.string_types)):
```

Comment:
Yes I think this case should be covered. I have used six.integer_types instead. It covers both `long` and `int` type.